UI/window-basic-main.cpp

```diff
@@ -7842,7 +7846,15 @@ void OBSBasic::on_actionEditTransform_triggered()
 
 void OBSBasic::on_actionCopyTransform_triggered()
 {
+	OBSSceneItem item = GetCurrentSceneItem();
+
+	obs_sceneitem_defer_update_begin(item);
+	obs_sceneitem_get_info(item, &copiedTransformInfo);
+	obs_sceneitem_get_crop(item, &copiedCropInfo);
+	obs_sceneitem_defer_update_end(item);
```

Comment:
I don't think we need to be calling `obs_sceneitem_defer_update_begin` / `obs_sceneitem_defer_update_end` here since the operations are read-only.

Reply:
I thought it might be safer, especially since the original implementation also had it, but on a second thought you're probably correct.